mule-packager/src/test/resources/real-app-target/pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.mule.fake</groupId>
+    <artifactId>complete-app</artifactId>
+    <version>1.0.0</version>
+    <name>A Mule Application fake</name>
+
+    <properties>
+        <javaVersion>1.8</javaVersion>
```

Comment:
not major but this branch of code is not meant to work with java 1.8 we'll support up to 1.7 only

Reply:
The pom is not actually used in the test so I just removed it